src/v2/guide/reactivity.md

```diff
@@ -118,3 +118,18 @@ Vue.component('example', {
   }
 })
 ```
+
+Since `$nextTick()` returns a promise, you can achieve the same as the above using the new [ES7 async/await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) syntax:
```

Comment:
AFAIK there's no "ES7." The official name is "ES2016," which I'd suggest using instead.

Reply:
Another PR should be fine I guess :)